test/lan_items_helpers_test.rb

```diff
@@ -166,3 +166,104 @@ def check_GetItemUdev(key, expected_value)
     end
   end
 end
+
+describe "LanItems#RemoveItemUdev" do
+  let(:rule) { { 0 => { "udev" => { "net" => ["KEY_TO_DELETE==\"VALUE\"", "OTHER_KEY"] } } } }
+  before(:each) do
+    Yast::LanItems.Items = rule
+    Yast::LanItems.current = 0
+  end
+
+  context "when the current item has an udev rule associated" do
+    it "removes the given key from the current rule if exists" do
+      Yast::LanItems.RemoveItemUdev("KEY_TO_DELETE")
+
+      expect(Yast::LanItems.GetItemUdevRule(0)).to eql(["OTHER_KEY"])
+    end
+
+    it "the current rule keeps untouched if the given key does not exist" do
+      Yast::LanItems.RemoveItemUdev("NOT_PRESENT_KEY")
+
+      expect(Yast::LanItems.GetItemUdevRule(0))
+        .to eql(["KEY_TO_DELETE==\"VALUE\"", "OTHER_KEY"])
+    end
+  end
+
+  context "when the current item doesn't have an udev rule associated" do
+    let(:rule) { { 0 => { "ifcfg" => "eth0" } } }
+
+    it "returns nil" do
+      expect(Yast::LanItems.RemoveItemUdev("KEY_TO_DELETE")).to eql(nil)
+    end
+  end
+end
+
+describe "#update_item_udev_rule!" do
+  let(:hwinfo) { { "dev_name" => "test0", "busid" => "00:08:00", "mac" => "01:02:03:04:05" } }
+  let(:udev_net) { ["ATTR{address}==\"01:02:03:04:05\"", "KERNEL==\"eth*\"", "NAME=\"test0\""] }
+  let(:rule) { { 0 => { "hwinfo" => hwinfo, "udev" => { "net" => udev_net } } } }
+
+  before do
+    Yast::LanItems.Items = rule
+    Yast::LanItems.current = 0
+    # The current item hasn't got a dev_port
+    allow(Yast::LanItems).to receive(:dev_port).and_return("")
```

Comment:
@mchf so by default I'm not exporting the dev_port and below we have a test that export it explicitly, the same is for `:mac` so, is one of this tests where you are missing some case? I have replaced the mock of `dev_port?` what was in use before the last changes.